packages/dx-react-grid/src/components/table-layout/virtual-table-layout.tsx

```diff
@@ -271,6 +274,21 @@ export class VirtualTableLayout extends React.PureComponent<PropsType, VirtualTa
     });
   }
 
+  unfocuseVirtualRowsChildren() {
+    const [lowerBoundary, upperBoundary] = this.props.viewport.rows;
+
+    const rowRefArray = Array.from(this.rowRefs, (entry) => {
```

Comment:
I usually use Object.entries/keys/values for this, assuming it should be faster to iterate maps than Array.from.
However, I may be wrong and it does not work with Map or is not actually faster...